packages/xeus/src/worker.ts

```diff
@@ -76,7 +66,7 @@ export class XeusWorkerLogger implements ILogger {
   private _channel: BroadcastChannel;
 }
 
-export class XeusRemoteKernel {
+export abstract class XeusRemoteKernel {
```

Comment:
I like that we're making this class abstract 👍 Thanks!

I don't want to be asking too much but can you add
```typescript
protected abstract mount(
    driveName: string,
    mountpoint: string,
    baseUrl: string,
    browsingContextId: string
  ): Promise<void>
```

to this base class too?

Reply:
Done, but it can't be `protected` because of https://github.com/jupyterlite/xeus/blob/2872782f4d4fe033ee479e074b7fffc96d374919/packages/xeus/src/coincident.worker.ts#L73